experiment/pretest.json

```diff
@@ -0,0 +1,72 @@
+{
+    "version": 2.0,
+    "questions": [{
+            "question": "“Torsion” is defined as _________",
+            "answers": {
+                "a": "Compressive force",
+                "b": "Type of friction",
+                "c": "Twisting",
+                "d": "Object at rest"
+            },
+            "explanations": {
+                "c": "Torsion is defined as the twisting of a specimen caused by a certain amount of torque. Compressive forces are forces that are applied to an object that results in the specimen getting compacted."
+            },
+            "correctAnswer": "c",
+            "difficulty": "beginner"
```

Comment:
Only beginner level questions available on pretest

Reply:
yes sir pranav said we could have 0 questions for some section if we want. and sme suggested that only the beginner level questions